projects/npm-tools/packages/npm-scripts/src/utils/getMergedConfig.js

```diff
@@ -153,17 +160,31 @@ function getMergedConfig(type, property) {
 			break;
 
 		case 'npmscripts': {
-			let presetConfig = {};
+			const rootDir = findRoot();
 
-			let userConfig = getUserConfig('npmscripts');
+			let rootDefaults;
 
-			// Use default config if no user config exists
+			try {
+				rootDefaults = require(path.join(
+					rootDir,
+					'npmscripts.config.js'
+				)).defaults;
```

Comment:
If `defaults` were ever `undefined` here, what would `deepMerge` do with it below? Gracefully ignore and continue?

Reply:
Probably fail .

![me coding](https://media.giphy.com/media/pFwRzOLfuGHok/giphy.gif)